includes/Settings.php

```diff
@@ -231,6 +236,17 @@ public function register(): void {
 			]
 		);
 
+		register_setting(
+			self::SETTING_GROUP,
+			self::SETTING_NAME_MGID_WIDGET_ID,
+			[
+				'description'  => __( 'MGID Widget ID', 'web-stories' ),
+				'type'         => 'integer',
+				'default'      => 0,
```

Comment:
```suggestion
				'default'      => null,
```

Let's change the default to `null` so that you're not greeted with a zero on the settings page initially:

<img width="691" alt="Screenshot 2023-06-14 at 17 35 33" src="https://github.com/GoogleForCreators/web-stories-wp/assets/841956/4af4f4f7-5eba-4404-9229-645ab8f9abb1">

That means updating the `Google\Web_Stories\Tests\Integration\Settings::test_on_plugin_uninstall()` test as well.